themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties

```diff
@@ -16,6 +16,7 @@ kerberosNotConfiguredTitle=Kerberos non configur\u00e9
 bypassKerberosDetail=Si vous n''\u00eates pas connect\u00e9 via Kerberos ou bien que votre navigateur n''est pas configur\u00e9 pour la connexion via Kerberos.  Veuillez cliquer pour vous connecter via un autre moyen.
 kerberosNotSetUp=Kerberos n''est pas configur\u00e9. Connexion impossible.
 registerTitle=S''enregistrer
+loginAccountTitle=Connectez-vous à votre compte
```

Comment:
Accentuated chars should be replaced by their unicode representation (in this case \u00e0 instead of à)

Reply:
I was aware of that and I missed it anyway :sweat_smile: